osmnx/simplify.py

```diff
@@ -71,13 +71,13 @@ def is_endpoint(G, node, strict=True):
 
         # add all the edge OSM IDs for incoming edges
         for u in G.predecessors(node):
-            for key in G.edge[u][node]:
-                osmids.append(G.edge[u][node][key]['osmid'])
+            for key in dict(G[u][node]).keys():
```

Comment:
Same as above: could replace ```G.edge``` with ```G.adj``` everywhere in the old code. You can also use ```G[u][node]``` in place of ```G.edge[u][node]``` if you prefer.

Reply:
@dschult thanks Dan, very helpful.